env_template.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+export BOOTSTRAP_SERVER= ...
+export CLUSTER_API_KEY= ...
+export CLUSTER_API_SECRET= ...
+export SR_ENDPOINT= ...
+export SR_API_KEY= ...
+export SR_API_SECRET= ...
+export SCHEMA_REGISTRY_BASIC_AUTH_USER_INFO=" ... : .... "
+export SCHEMA_REGISTRY_URL="https://.... "
+
+export SPARK_HOME=/opt/homebrew/Cellar/apache-spark/3.5.2/libexec
```

Comment:
This seems to be mac os specific. Maybe also make it an argument or `...`

Reply:
since that's an template we can add a note that's for OSX environments. Linux / Windows would be different. Or @kamir renames the file to osx_env_template.sh ;) 4

Reply:
That would be the case, if it was like this for all OSX users. I think this shouldn't be versioned at all, as it makes a lot of assumptions on ones setup. This can easily be excluded and we could prepare a .env.example file that holds values like this as "guardrails". What do you think @kamir @2pk03?